server/spec/sku_level_enable_override_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'spec_helper'
+require 'candlepin_scenarios'
+
+describe 'SKU Level Enable Override' do
+  include CandlepinMethods
+  include CertificateMethods
+
+  before(:each) do
+    @owner = create_owner random_string('test_owner')
+    @org_admin = user_client(@owner, random_string('user'))
+    @env = @org_admin.create_environment(@owner['key'], 'testenv1',
+      "My Test Env 1", "For test systems only.")
+
+    @content1 = create_content
+    @content2 = create_content
+    content_list = @content1.id + "," + @content2.id
+    @product1 = create_product(nil, nil,  {:attributes => { :content_override_active => content_list }})
+    @product2 = create_product(nil, nil,  {:attributes => { :content_override_disabled => content_list }})
```

Comment:
The attribute names don't quite match, active/inactive, enabled/disabled, but active/disabled doesn't seem right.